concord-consortium/collaborative-learning
# 186610287 One Data Legend

PT Stories:
https://www.pivotaltracker.com/story/show/186610287
https://www.pivotaltracker.com/story/show/186718726

This PR moves the x axis selection for data plots from directly under the x axis into the data plot section of the legend. The x axis selection should still be in the same old location for units that don't use the standard legend (such as `mothed`). Additionally:
- Styling for the whole legend has been cleaned up.
- An unlink button has been added to the variable section of the legend.
- Moves the variable section of the legend above the data section.

QA considerations:
- I tried to clean up the styling, but there's a lot unclear to me from the spec, specially for graphs of different sizes. I think it would make sense to submit this as is and get a round of feedback from design, which can be incorporated into this styling story: https://www.pivotaltracker.com/story/show/184983429
- The height of the legend is still inaccurate. It should be improved in the legend plugin story, when the legend code will be streamlined: https://www.pivotaltracker.com/story/show/186345543
